docs/Installing.asciidoc

```diff
@@ -386,9 +386,18 @@ In the +/etc/openqa/workers.ini+
 [source,ini]
 --------------------------------------------------------------------------------
 [global]
-CACHEDIRECTORY = /var/lib/openqa/cache
+HOST=http://webui
+CACHEDIRECTORY = $cache_location
+CACHELIMIT = 50 # This is currently noop. Cache supports limiting, but is not enabled.
```

Comment:
So why mention it in the docs when it has no effect?

Reply:
Because It's a premium feature :) You need to request it, and you will be charged the equivalent the latest Tesla model 3 to implement the two lines of code :P

Reply:
Maybe insert a paypal link then ;)